changelogs/fragments/1036-apf-try-except.yml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - zos_apf - Modified Try Except Pass block when operation = list so that now the module
```

`when operation = list` can become `when operation is a list`

---

present tense is better for minor_changes. 

- zos_apf - Modifies the try except pass block when operation is a list so that now the module...

---

agree, I always post the ansible developtment guide and refer to this but this time I forgot 😅